edge-modules/MetricsCollector/Program.cs

```diff
@@ -32,7 +33,7 @@ static async Task<int> MainAsync()
             try
             {
                 moduleClient = await ModuleClient.CreateFromEnvironmentAsync(transportSettings);
-                Dictionary<string, string> additionalTags = await GetAdditionalTagsFromTwin(moduleClient);
+                Option<SortedDictionary<string, string>> additionalTags = await GetAdditionalTagsFromTwin(moduleClient);

```

Comment:
I think it is ok to return empty SortedDictionary, right?

Reply:
I thought using Option was better because then one would have to check the contents of the collection, which is a little less obvious. Option lets the reader know the intentions all the way through the code. Let me know if you disagree.